analysis/topEFT/make_jsons.py

```diff
@@ -464,26 +464,34 @@
         "histAxisName": "ttZ_centralUL16",
         "xsecName": "TTZToLLNuNu_M_10",
     },
-    "UL16_tZq" : {
-        "path" : "/tZq_ll_4f_ckm_NLO_TuneCP5_PSweights_13TeV-amcatnlo-pythia8/RunIISummer16NanoAODv7-PUMoriond17_Nano02Apr2020_102X_mcRun2_asymptotic_v8-v1/NANOAODSIM",
-        "histAxisName": "tZq_centralUL17",
-        "xsecName": "tZq",
-    },
     "UL16_tttt" : {
         "path" : "/TTTT_TuneCP5_13TeV-amcatnlo-pythia8/RunIISummer20UL16NanoAODv9-106X_mcRun2_asymptotic_v17-v2/NANOAODSIM",
         "histAxisName": "tttt_centralUL16",
         "xsecName": "tttt",
     },
 }
 
+central_UL16_tZq_dict = {                                                                                                               #tZq is the only central sig sample in hadoop for now
+    "UL16_tZq" : {
+        "path" : "/tZq_ll_4f_ckm_NLO_TuneCP5_13TeV-amcatnlo-pythia8/RunIISummer20UL16NanoAODv9-106X_mcRun2_asymptotic_v17-v1/NANOAODSIM",
+        "path_local" : "/store/mc/RunIISummer20UL16NanoAODv9/tZq_ll_4f_ckm_NLO_TuneCP5_13TeV-amcatnlo-pythia8/NANOAODSIM/106X_mcRun2_asymptotic_v17-v1",
+        "histAxisName": "tZq_centralUL17",
+        "xsecName": "tZq",
+    },
+}
+
```

Comment:
Can you merge this into the main `central_UL16_dict` dict?

Reply:
@bryates, the reason I split this up is cause this is the only central sig sample that has been moved to ND hadoop. In [these lines](https://github.com/TopEFT/topcoffea/blob/update_tZq_WZTo3LNu_samples/analysis/topEFT/make_jsons.py#L1377-L1390), the last four lines will run the script on these four new dicts for tZq sample.

Reply:
Perhaps @kmohrman could comment on this?

Reply:
I am fine with having a separate dictionary for the new v9 local copies of central samples, though if @bryates you'd strongly prefer that they be included in the main dictionaries, that's ok with me too. If they are going to be a separate dictionary, I would probably prefer the naming to indicate why they're separate (currently the name of the dict just indicates the sample name, but if you included v9 or local or something, it might help to highlight why this sample is different from the old ones).

Reply:
That's fine. I guess I was thinking we had others, but if this is the first batch of local central samples, it makes sense.